lib/redux/middleware/sentry_sdk_middleware.dart

```diff
@@ -8,6 +9,18 @@ class SentrySdkMiddleware extends MiddlewareClass<AppState> {
 
   @override
   dynamic call(Store<AppState> store, dynamic action, NextDispatcher next) async {
+    if (action is RehydrateSuccessAction) {
+      if (action.sentrySdkEnabled) {
+        _enableSentrySdk();
+      }
+    }
+    if (action is SentrySdkToggleAction) {
+      if (action.enabled) {
+        _enableSentrySdk();
+      } else {
+        _disableSentrySdk();
+      }
```

Comment:
if this feature can be enabled/disabled multiple times per App's lifecycle, I'm not entirely sure this works, eg sentry-java, sentry-native and sentry-cocoa are still fully active, also, Sentry.init is designed to call only once during App's lifecycle, this could uncover issues that I'm unaware of, better to test it out

Reply:
Ok, so would we be able to at least once init from settings. Or close an already initialized project?

Reply:
yeah, that would be better, or of course, test the current implementation, enable/disable multiple times and see if it enables again, if it disables again, if the signal handlers capture events properly etc, its not tested for sure.

https://develop.sentry.dev/sdk/unified-api/#static-api

check the `init` block